dashboard/config/scripts/20-hour.script

```diff
@@ -93,17 +93,19 @@ concepts 'if_else', 'loop_until'
 level_concept_difficulty '{"sequencing":5,"repeat_until_while":4,"conditionals":5}'
 level 'blockly:Maze:2_19'
 
-lesson 'Computational Thinking'
+lesson 'Computational Thinking', display_name: 'Computational Thinking'
+skin 'pvz'
 concepts 'sequence'
-level_concept_difficulty '{}'
```

Comment:
These changes are coming in because these script files have not been touched since we have changed the script DSL syntax. I checked that the scripts that had these changes still work the same

Reply:
In the future it could be helpful to run something in advance like `Script.all.each(&:save!)` to get all of the scripts to write out their backlogged changes to their script files, so that we can have a clearer picture of what's new as part of your code change.

Reply:
Do you want me to make a pr ahead of this one that does that so you can see the difference?

Reply:
no, this is fine for this time. this is just an option to keep in mind for next time.